.github/workflows/release.yaml

```diff
@@ -323,13 +323,21 @@ jobs:
       - uses: actions/setup-node@v2.5.1
         with:
           node-version: "16"
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.17"
+      - run: go install sigs.k8s.io/bom/cmd/bom@v0.2.0
+      - run: go install github.com/spdx/spdx-sbom-generator/cmd/generator@v0.0.13
+      - run: mkdir -p dist
+      - run: generator -o dist -p .
+      - run: generator -o dist -p ui
+      - run: bom generate --image quay.io/argoproj/workflow-controller:${GITHUB_REF##*/}
+      - run: bom generate --image quay.io/argoproj/argocli:${GITHUB_REF##*/}
+      - run: bom generate --image quay.io/argoproj/argoexec:${GITHUB_REF##*/}
```

Comment:
You are missing the `-o` flag here to create the spdx files in the `dist` folder